server/src/com/scraniel/auctioneero/AuctionManager.java

```diff
@@ -54,30 +68,57 @@ public AuctionResponse bidOnItem(UUID itemId, UUID bidderId, float bid)
 			// Look up SQL for this, probably UPDATE or ALTER
 			// END;
 			//
-			message = String.format("Your bid of {0} is the new highest bid", bid);
+			message = String.format("Your bid of %f is the new highest bid", bid);
 		}
 		
-		return new AuctionResponse(isSuccessfulBid, message);
+		throw new NotYetImplementedException();
 	}
 	
 	// Puts a new item up for auction; do we care about conflicts? Probably not, maybe within user's own items
 	// Need to scrub name parameter to protect against injection
 	public AuctionResponse addItem(String name, String description, UUID userId, float startingBid, Timestamp expiry)
 	{
-		// 1. Get user's items, make sure there are none with same name (this step is possibly unnecessary)
+		// 1. Get user's items, make sure there are none with same name (this step is possibly unnecessary, and for now is skipped)
 		// Something like:
 		// SELECT count(*) FROM items WHERE user_id = '{userId}' and name = {name}
+		AuctionItem item = new AuctionItem();
```

Comment:
a constructor which allows you to initialize all of those fields my be a good idea.
As an alternative, I think in java/Android, they like to use the builder pattern for that
AuctionItem item = new AuctionItem().
.setName(name)
.ownerId(userId)...

Reply:
Yes, that's the plan. I wasn't sure if Hibernate _necessitated_ POJO, but turns out it doesn't so I will fix this.